src/lib/swapper/swappers/CowSwapper/cowGetTradeTxs/cowGetTradeTxs.ts

```diff
@@ -11,10 +11,11 @@ import { cowService } from 'lib/swapper/swappers/CowSwapper/utils/cowService'
 
 export async function cowGetTradeTxs(
   deps: CowSwapperDeps,
+  network: string,
```

Comment:
Similarly not sure about passing `network` around - we get a `ChainId` we can use from deps, could call the `getNetwork` util / use the mapping if you go that route, instead of passing some additional param around